### PR TITLE
Cleanup Norwegian translation file

### PR DESCRIPTION
**Pull request by Tbaut**
I'm not quite fluent, but this doesn't look very Norwegian :)

### CODE REVIEW

**Review by asnaith — APPROVED** [review]

File: packages/files-ui/src/locales/no/messages.po [hunk 1]
```diff
@@ -86,7 +86,7 @@ msgid "Anyone with the link can:"
 msgstr ""
 
 msgid "Approve"
-msgstr "Godkjenn<<<<<<< HEAD<<<<<<< HEAD"
+msgstr ""
```
Comment (FSM1) [inline packages/files-ui/src/locales/no/messages.po]:
```suggestion
msgstr "Godkjenn"
```

File: packages/files-ui/src/locales/no/messages.po [hunk 2]
```diff
@@ -311,13 +311,13 @@ msgid "Drop to upload files"
...
 msgid "Edit payment method"
-msgstr "<<<<<<< HEAD"
+msgstr ""
 
 msgid "Edit plan"
 msgstr ""
 
 msgid "Email is required"
-msgstr "E-post kreves=======>>>>>>> origin/dev"
+msgstr ""
```
Comment (FSM1) [inline packages/files-ui/src/locales/no/messages.po]:
```suggestion
msgstr "E-post kreves"
```

**Review by FSM1 — DISMISSED** [review]
2 actual translations seem to have gotten caught in the crossfire 😅

**Comment by asnaith** [comment]
> 2 actual translations seem to have gotten caught in the crossfire 😅

Good catches there @FSM1. 

It made me think - would it be worth us adding a couple of rules to the linter "HEAD<<<<<<<" text etc?

File: packages/files-ui/src/locales/no/messages.po [hunk 3]
```diff
@@ -1064,13 +1064,13 @@ msgid "You get access to these features right now."
 msgstr ""
 
 msgid "You haven't set a username yet."
-msgstr "Du har ikke satt noe brukernavn enda.<<<<<<< HEAD"
+msgstr ""
```
Comment (Tbaut) [inline packages/files-ui/src/locales/no/messages.po]:
```suggestion
msgstr "Du har ikke satt noe brukernavn enda."
```

**Comment by Tbaut** [comment]
> would it be worth us adding a couple of rules to the linter "HEAD<<<<<<<" text etc?

I think it's overkill. These were mistakes from big conflicts with the previous billing branch. Also I'm not sure how to actually ask the linter about these.

**Comment by asnaith** [comment]
> > would it be worth us adding a couple of rules to the linter "HEAD<<<<<<<" text etc?
> 
> I think it's overkill. These were mistakes from big conflicts with the previous billing branch. Also I'm not sure how to actually ask the linter about these.

@Tbaut Okay cool, just thought I'd throw the idea out there 👍

**Pull request merged**